services/API-service/src/api/event/event.service.ts

```diff
@@ -1086,19 +1107,20 @@ export class EventService {
       endDate: LessThan(lastUploadDate.timestamp), // If the area was not prolonged earlier, then the endDate is not updated and is therefore less than the lastUploadDate
       adminArea: { countryCodeISO3 },
       disasterType,
+      userTrigger: false, // do not close event triggered by user
```

Comment:
Hmm, I know where this comes from. But it needs to close at some point right? This way it can never disappear.

Reply:
@gulfaraz thought about this more, and I think we should revert this for now. To bring it back in AB#32053, where we can apply filtering such that event with userTrigger:true does close if lastUploadDate>endDate, and otherwise not. Agree?

Reply:
done https://github.com/rodekruis/IBF-system/pull/2036/commits/421222f1e75559ff6f4b248794ba072b0afc158e